spectre-meltdown-checker.sh

```diff
@@ -586,8 +586,8 @@ if [ "$opt_live" = 1 ]; then
 	elif [ -e /sys/kernel/debug/x86/pti_enabled ]; then
 		# RedHat Backport creates a dedicated file, see https://access.redhat.com/articles/3311301
 		kpti_enabled=$(cat /sys/kernel/debug/x86/pti_enabled 2>/dev/null)
-	elif dmesg | grep -Eq 'Kernel/User page tables isolation: enabled|Kernel page table isolation enabled'; then
```

Comment:
I think trying dmesg, then if not found (because the buffer lost the boot messages), try `/var/log/dmesg` as you proposes, maximises the chances to find the message if it's there. Hence, can you keep those two lines ?